src/js/utils/StringUtil.js

```diff
@@ -66,6 +66,23 @@ const StringUtil = {
     return string.charAt(0).toUpperCase() + string.slice(1, string.length);
   },
 
+  humanizeArray: function (array, serialComma=true) {
```

Comment:
Thanks for pointing this out - I didn't realise `arrayToJoinString` existed!

Having poked around the code a bit, it looks like `arrayToJoinedString` is only used to join configuration parameters in the `ConfigurationView`. Do you reckon the intents are similar enough to warrant joining them?